conf/boards/xvert_1.0.makefile

```diff
@@ -0,0 +1,71 @@
+# Hey Emacs, this is a -*- makefile -*-
+#
+# stm32f37_vortex_chibios.makefile
+#
+#
+
+BOARD=xvert
+BOARD_VERSION=1.0
+BOARD_DIR=$(BOARD)/chibios/v$(BOARD_VERSION)
+BOARD_CFG=\"boards/$(BOARD_DIR)/board.h\"
+
+ARCH=chibios
+$(TARGET).ARCHDIR = $(ARCH)
+
+RTOS=chibios
+
+## FPU on F3
+
+##TODO: there is some bug when using the hard FPU unit!!
+USE_FPU=no
```

Comment:
can you file a bug report at ChibiOS forum about it? Giovanni is typically fairly responsive.

Reply:
It's probably something with wrong settings for the minimal float number. Was planning to look into it a bit further myself later.